c-bindings-gen/README.md

```diff
@@ -0,0 +1,14 @@
+LDK C Bindings Generator
+========================
+
+This program parses a single-file-lib.rs Rust crate's AST passed in on stdin and generates a second
+crate which is C-callable (and carries appropriate annotations for cbindgen). It is usually invoked
+via the `genbindings.sh` script in the top-level directory, which converts the lightning crate into
+a single file with a call to
+`RUSTC_BOOTSTRAP=1 cargo rustc --profile=check -- -Zunstable-options --pretty=expanded`.
```

Comment:
Were any changes resulting from https://github.com/rust-bitcoin/rust-lightning/pull/802#discussion_r580501270 incorporated when splitting into this PR? Looks like it may have been missed, but the diff is no longer available there.

Reply:
Heh, I'd responded to it, but only partially.